hooks/useNFTV3.ts

```diff
@@ -0,0 +1,208 @@
+import {
+  useState,
+  useEffect,
+  useRef,
+  MutableRefObject,
+  SetStateAction,
+  Dispatch
+} from 'react';
+import { Connection, PublicKey, clusterApiUrl } from '@solana/web3.js';
+import {
+  resolveToWalletAddress,
+  getParsedNftAccountsByOwner
+} from '@nfteyez/sol-rayz';
+import { ConnectedWallet, useSolana } from '@saberhq/use-solana';
+import { programs } from '@metaplex/js';
+import { MetadataKey } from '@nfteyez/sol-rayz/dist/config/metaplex';
+import honeyGenesisBee from '/public/images/imagePlaceholder.png';
+
+interface NFTArrayType {
+  [index: string]: Array<NFT>;
+}
+
+const defaultNFT: NFT = {
+  name: '',
+  updateAuthority: '',
+  image: '',
+  creators: [],
+  tokenId: '',
+  mint: ''
+};
+
+export const defaultNFTImageUrl =
```

Comment:
Validate which default img we want - preferably a static generated one from our public folder